_includes/code/graphql.additional.py

```diff
@@ -1,12 +1,21 @@
-import weaviate
+# TESTS NEED TO BE UPDATED
+# START-ANY
 import json
+import os
+import weaviate
+import weaviate.classes as wvc
+from weaviate.collections.classes.grpc import Sort
+# END-ANY
```

Comment:
Can we add this please?
```python

client = weaviate.connect_to_local()

```

This section renders on every snippet. So if you look at the preview - the first snippet will look like this:

```python
import weaviate
import weaviate.classes as wvc
import os

client = weaviate.connect_to_local()

articles = client.collections.get("Article")

response = articles.query.near_text(
    query="fashion",
    limit=5,
    return_metadata=wvc.query.MetadataQuery(distance=True)
)

for o in response.objects:
    print(o.metadata.distance)  # Inspect metadata
    print(o.uuid)  # Inspect UUID (returned by default)
    print(o.properties)  # Inspect returned objects
```

Reply:
Sure, is START-ANY new? We don't seem to use it very much. (I don't remember it from the howto pages.)

Reply:
I added it a month or so ago as some marker lines were getting quite long.

Reply:
Updated